dietpi/dietpi-software

```diff
@@ -11174,6 +11213,54 @@ _EOF_
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# User
+			local usercmd='useradd -rMU'
+			getent passwd ffsync &> /dev/null && usercmd='usermod -a'
+			$usercmd -G dietpi -d /opt/ffsync -s $(command -v nologin) ffsync
```

Comment:
Users home dir should be an existing one, I guess? If config file can be moved to outside the program/install dir, I would use this, else the program/install dir. If this will be `/opt/ffsync` (as suggested above), this is fine already, of course :wink:.